Project2015To2017/ProjectConverter.cs

```diff
@@ -11,9 +11,10 @@
 
 namespace Project2015To2017
 {
-	public class ProjectConverter
+	public static class ProjectConverter
```

Comment:
This is a breaking change, so I'm going to skip this one in the merge

Reply:
Is it? I mean, do we care about binary compatibility or usage compatibility? The use is the same, no changes needed.

Reply:
You're right, it doesn't matter in this case.